spec/fixtures/locations.json

```diff
@@ -14,6 +14,7 @@
       "properties": {
         "title": "Belfast Citizens Advice Bureau",
         "address": "Merrion Business Centre, 58 Howard St, Belfast, BT1 6PJ",
+        "booking_centre": "928543c7-e8f0-45b2-a300-c2f996b007d9",
```

Comment:
Took me longer than anticipated to write that last comment ;)

So, the plan is to start using `booking_location_id` in this app *now* while the GeoJSON still contains `booking_centre` which will be retired in due course.